### DISCUSSION TITLE
Running error

### DISCUSSION THREAD

**Discussion by bvh97** [CoExpNetViz/CoExpNetViz#22] [Q&A]
Hi all,

I am trying to run the CoExpNetViz with my dataset containing ~30000 genes. However, it always shows the error below after running 30 mins. (I have attached the file below).

Is it because of my pc having not enough memory for it? Or is there any other problems? 

I would really appreciate your help to solve the issues.

Have a good day! Thank you so much!
![image](https://user-images.githubusercontent.com/113940698/191100311-b3d24f4c-a9f8-43ab-aed8-0e0d68f87660.png)

**Comment by hannahdiels** [comment]
Hi, how many baits are you using and how many conditions (i.e. columns minus the gene column) does your expression matrix have? How much free memory do you have left with cytoscape open before submitting the form to create the co-expression network?
